### ISSUE TITLE
[BUG] When using archives, URL calculation is not using the correct path

### ISSUE THREAD

**Issue by tombrunet** [IBMa/equal-access#329]
## What tool(s) are you using?
- `accessibility-checker`

## Description
When running the Node checker with an archive other than `latest` or `preview`, the scan fails. This is due to a defect in the URL calculation. For instance, when specifying archive 10Feb2021, the checker uses the URL `/rules/archives/10Feb2021/js/ace-node.js`

It should be looking into the archives definition to find the path property:
```
    "path": "/archives/2021.02.10",
```

The correct path is therefore `/rules/archives/2021.02.10/js/ace-node.js`.

**Comment by tombrunet** [comment]
Merged - will deploy in next release
